app/Http/Controllers/UserCreditsController.php

```diff
@@ -27,7 +27,7 @@ public function store(Request $request, $id){
         return redirect()->action('GameController@edit', [$id]);
     }
 
-    public function destroy($id, $credit_id){
+    public function destroy($id, $credit_id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->